network/network_windows_test.go

```diff
@@ -430,3 +430,77 @@ func TestNewNetworkImplHnsV2ForBackendNIC(t *testing.T) {
 		t.Fatal("HNS network is created with BackendNIC interface")
 	}
 }
+
+// mock hns network creation and deletion for DelegatedNIC
+func TestNewAndDeleteNetworkImplHnsV2ForDelegated(t *testing.T) {
```

Comment:
i didn't ask to validate create and delete hns network.. my ask when CNI delete was invoked, the unit test should make sure delete netwrok was invoked if nic type is delegated or acclnet